core/src/main/java/com/sequenceiq/cloudbreak/job/nodestatus/NodeStatusCheckerConfig.java

```diff
@@ -12,7 +12,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NodeStatusCheckerConfig.class);
 
-    @Value("${nodestatuschecker.intervalsec:1200}")
+    @Value("${nodestatuschecker.intervalsec:1800}")
```

Comment:
why the increase here?

Reply:
we do not need these data too often, so 30 mins looked better choice